### ISSUE TITLE
HTML inside script tags is turned invalid

### ISSUE THREAD

**Issue by ozupey** [voku/HtmlMin#29]
#### What is this feature about (expected vs actual behaviour)?
Any HTML that's inside a `<script type=text/html>` tag is turned invalid. Similar to issue #26. Seems like https://github.com/voku/simple_html_dom/commit/e6e597e2bb774753e49ed6e8101ff7079691858b#diff-c9aceb5e54e67d5f19805df3f679fb1bR389 only works on double quotes or single quotes (`type="text/html"` or `type='text/html'`) rather than no quotes at all (`type=text/html`).

```
string(81) "<script type=text/html><p>Foo

	<div class="alert alert-success">
		Bar</script>"
```

#### How can I reproduce it?
```
$html = '
<script type=text/html>
	<p>Foo</p>

	<div class="alert alert-success">
		Bar
	</div>
</script>
';

$htmlMin = new voku\helper\HtmlMin();
$result = $htmlMin->minify($html);

var_dump($result);
```
#### Does it take minutes, hours or days to fix?
Unsure

#### Any additional information?
Using 3.1.3 installed via composer on PHP 7.2.10.

**Comment by voku** [comment]
Thanks for the bug report, fixed in version 3.1.6

**Comment by ozupey** [comment]
It looks like any occurrence of `{{` will cancel minification. 

It seems to be on purpose, but I'm not sure why: https://github.com/voku/simple_html_dom/blob/master/src/voku/helper/HtmlDomParser.php#L947

If I disable that line, it minifies just fine and I'm not seeing any negative effects.

```
<script type="text/html">
	{{foo}}

	{{bar}}

	{{hello}}
</script>
```

Will result in:
```
string(63) "<script type=text/html>	{{foo}}

	{{bar}}

	{{hello}}
</script>"
```

**Comment by ozupey** [comment]
It would be nice if there was at least an option to force minification of these special tags, perhaps disabled by default. :)

**Comment by voku** [comment]
Hi and thanks for the bug report (again) :+1:  ... fixed in version 4.4.0. Here is an example: https://github.com/voku/HtmlMin/commit/9a88a5c737a249b0ff7e5b3a17f0111ce8f13999#diff-468b8a043eb54ed31edbee53141c1512R769
